transport_and_format.asciidoc

```diff
@@ -312,81 +312,83 @@ include::examples/transport_and_format_get_token_list_example.json[]
-It might be good practice to always provide the 'OCPI-from-' headers in a response,
-just to be complete and to give a requesting party to change to validate if the request was routed correctly by the Hub.
+When a party responds to a request (via a Hub), and it has been a request that contained the 'OCPI-to-' header,
+the 'OCPI-from-' header MAY be omitted. The requesting party knows to which party the request was sent.
+It might be good practice to always provide the 'OCPI-from-' header in a response though,
+just to be complete and to give the requesting party a chance to validate if the request has been routed correctly by the Hub.
 
 
 [[transport_and_format_message_routing_broadcast_push]]
 ===== Broadcast push
 
 For simplicity, connected clients might PUSH (POST, PUT, PATCH) information to all connected clients with an "opposite role",
 CPO pushing information to all eMSPs and NSPs. eMSP pushing information to all CPOs.
-(the role "Other" is seen as an eMSP type of role, so broadcast push from a CPO is also send to Other,
-messages from "Other" are send to CPOs, not the eMSPs.)
+(The role "Other" is seen as an eMSP type of role, so broadcast push from a CPO is also sent to "Other".
+Messages from "Other" are only sent to CPOs and not to eMSPs though.)
 
-Broadcast push might be very useful to push information like Locations or Tokens to all parties,
+Broadcast pushing data through a Hub might be very useful to share information like Locations or Tokens with all parties
 connected to the Hub that have implemented the corresponding module.
-This means only 1 request to the Hub, not having to worry about the number of connected clients.
+This means only one request to the Hub will be necessary, as all connected clients will be served by the Hub.
+In other words: the Hub broadcasts received information to all connected clients.
 
-To send a Broadcast push, the client uses to Hubs party-id and country-code in the 'OCPI-to-' headers.
-The Hub parses the request and sends a response to the client, with optionally its own party-id and country-code in the 'OCPI-from-' headers.
-The Hub then sends the push to any client implementing the applicable module, using its own party-id and country-code in the 'OCPI-from-' headers.
-The client receiving a push from a Hub (with the Hubs information in the 'OCPI-from-' headers) will respond to
-this push with the Hubs party-id and country-code in the 'OCPI-to-' headers.
+To send a broadcast push, the client uses the party-id and country-code of the Hub in the 'OCPI-to-' header.
+The Hub parses the request and sends a response to the client, which optionally contains its own party-id and country-code in the 'OCPI-from-' header.
+The Hub then sends the pushed data to any client implementing the corresponding applicable module, using its own party-id and country-code in the 'OCPI-from-' header.
+The client receiving a PUSH from a Hub (with the Hubs information in the 'OCPI-from-' header) will respond to
```

Comment:
Why do you choose to use all caps: PUSH here?

Reply:
I changed a few more occurences of it which are not shown in the diffs by GitHub for whatever reason:
> eMSPs who do not support PUSH mode need to call GET on the CPOs Locations endpoint to receive the new object.

> It is advised to use the PUSH direction from CPO to eMSP during normal operation [...].

> This feature might be used by an eMSP that wants information about a specific Location, but has not implemented the eMSP Locations interface (i.e. cannot receive PUSH). 

To me it simply feels right to _somehow_ pronounce that we are talking about PUSH/PULL. We implicitely do the same when we write `GET`, `POST`, `PATCH`, ... so why not in this case?

Other examples are _Broadcast Push_ and _Open Routing Request_ where we pronounce a specific term by writing them explicitely in a different way. In above examples, _Push Mode_ could be used as such term. But actually it would be rather easy to avoid the word `push` entirely:

> eMSPs who do not support ~~PUSH mode~~ **receiving data** need to call GET on the CPOs Locations endpoint to receive the new object.

> It is advised to use the ~~PUSH~~ direction from CPO to eMSP during normal operation [...].

> This feature might be used by an eMSP that wants information about a specific Location, but has not implemented the eMSP Locations interface (i.e. cannot receive ~~PUSH~~ **data**). 

However, the second example would be somewhat arbitrary.

Reply:
GET, POST, PATCH etc are HTTP Methods that are always in all caps. 

Push and Pull are not very familiar terms for implementers of OCPI, removing the terms would not help them.

Reply:
I updated this again in cf5e3de822996fa550a47476d301bbfa23c21d03 for all files and occurences of _push_ and _pull_. I used `Push`/`Pull` when we talk about `Push mode` or `Push direction`, but `push[(ing|es|ed)]` everywhere else.